status/condition_set.go

```diff
@@ -99,12 +106,19 @@ func (c ConditionSet) IsTrue(conditionTypes ...string) bool {
 	return true
 }
 
+func (c ConditionSet) IsNormalCondition(t string) bool {
```

Comment:
I really liked the dependent/independent terms you mentioned the other day.
```suggestion
func (c ConditionSet) IsDependentCondition(t string) bool {
```